app/models/assignment.rb

```diff
@@ -31,16 +31,16 @@ class Assignment < ApplicationRecord
   scope :no_end, (-> { where(period_end: nil) })
   scope :has_end, (-> { where.not(period_end: nil) })
 
-  scope :ended, (-> { where('period_end < ?', Time.zone.now) })
+  scope :ended, (-> { where('period_end < ?', Time.zone.now.to_date) })
```

Comment:
Wanted to say `Date.today` but that doesn't use the time zone from Rails. But there's also `Time.zone.today` which returns a `Date`

Reply:
Ah, thats a great info. Rubocop was always bashing me if I did `Date.today`, so I used `Time.zone.now.to_date`, but probably RuboCop might like `Time.zone.today` better ;) Thanks.